src/main/kotlin/app/revanced/patches/youtube/layout/personalinformation/resource/patch/HideEmailAddressResourcePatch.kt

```diff
@@ -26,10 +26,10 @@ class HideEmailAddressResourcePatch : ResourcePatch {
         SettingsPatch.PreferenceScreen.LAYOUT.addPreferences(
             SwitchPreference(
                 "revanced_hide_email_address",
-                StringResource("revanced_hide_email_address_title", "Hide the email address"),
+                StringResource("revanced_hide_email_address_title", "Hide email address"),
                 false,
-                StringResource("revanced_hide_email_address_summary_on", "Email address is hidden"),
-                StringResource("revanced_hide_email_address_summary_off", "Email address is visible")
+                StringResource("revanced_hide_email_address_summary_on", "Email is not shown in account switcher"),
```

Comment:
This information should be added to the patch description instead of here, in favour of consistent summaries across all toggles.

Reply:
I changed it to:
`"Hide email in account switcher"
"Email address is not shown"
"Email address is shown"`